sound/soc/sof/ipc3-pcm.c

```diff
@@ -380,4 +380,5 @@ const struct sof_ipc_pcm_ops ipc3_pcm_ops = {
 	.hw_free = sof_ipc3_pcm_hw_free,
 	.trigger = sof_ipc3_pcm_trigger,
 	.dai_link_fixup = sof_ipc3_pcm_dai_link_fixup,
+	.reset_hw_params_during_stop = true,
```

Comment:
"This will ensure that the stream tag will not be given up during the
STOP trigger for IPC4."

We removed hda_link_dma_cleanup() in the previous patch, so not following why we still need to do this.

Is this needed maybe on the host side for e.g. nocodec cases?

Reply:
@plbossart why only for nocodec? we need to preserve the host DMA stream tag for all cases isnt it?

Reply:
What I meant is that for the link DMA, the first patch makes sure that the stream_tag is not released.

What I was asking if whether this second patch is targeting the host DMA, and you've answered to that effect in your comment above.

So my suggestion is to clarify that the second patch will only have effects on host DMAs.

Reply:
done